book/src/development/infrastructure/release.md

```diff
@@ -13,12 +13,11 @@ steps to create a Clippy release.
 4. [Tag the stable commit](#tag-the-stable-commit)
 5. [Update `CHANGELOG.md`](#update-changelogmd)
 
-_NOTE: This document is for stable Rust releases, not for point releases. For
-point releases, step 1. and 2. should be enough._
+> _NOTE:_ This document is for stable Rust releases, not for point releases. For
+> point releases, step 1. and 2. should be enough.
 
 [Rust Forge]: https://forge.rust-lang.org/
 
-
 ## Remerge the `beta` branch
```

Comment:
For the scripts in this section, it would be good to know in which repo they should be executed

Reply:
I think I always mention this. But it seems to not be clear enough. I try to improve on that.

Reply:
This was the only case, where it wasn't quite clear :)